### DISCUSSION TITLE
URL parameter to attach a webpage?

### DISCUSSION THREAD

**Discussion by mariomaz87** [open-webui/open-webui#8193] [Q&A]
Hi, I’m trying to create an iOS shortcut to pass the shared url to OpenWeb Ui and ask for a summary. Is there a url parameter that is the equivalent of “#url” in a chat? Thanks

**Comment by ubergarm** [comment]
> Hi, I’m trying to create an iOS shortcut to pass the shared url to OpenWeb Ui and ask for a summary. Is there a url parameter that is the equivalent of “#url” in a chat? Thanks

Took me forever to figure it out too. It is `#` lol...

1. Startup Open WebUI
2. You might have to enable web search but I'm not sure.
3. Type in `#https://emptyduck.com` or any URL with the `#` as the very first character and then wait for a moment.
4. Click on the box/button that appears above the chat window and it will fetch the html content.
5. Then you finally ask your question related to the content it pulled into context.

The documentation has been challenging for me, a simple tutorial video would be nice. Cheers!

**Reply by ubergarm** [reply]
Oh sorry, I have no idea about iOS... But this was using web page so probably would work through your browser?

**Reply by mariomaz87** [reply]
Hi, thanks for the feedback but I'm asking about an url parameter, so something to be done using the url bar directly and not in the UX.
